hyperdome_server/onion.py

```diff
@@ -580,13 +578,13 @@ def start_onion_service(self, port):
                 res = self.c.create_ephemeral_hidden_service(
                     {80: port}, await_publication=await_publication,
                     basic_auth=basic_auth, key_type=key_type,
-                    key_content=key_content)
+                    key_content=key_content, timeout=10)
```

Comment:
10 seconds may be too short for users with a high latency connection or poor node locality.

Reply:
Definitely agree. We can increase the value by a lot, but the issue is that onion connection fails very often, so we need to retry a lot; I want to figure out why that's happening.

Reply:
likely being addressed by #39